### ISSUE TITLE
Converted Switch File | HP, Gold, Attack & Defense Unchangeable

### ISSUE THREAD

**Issue by infinitytrain** [jonyluke/Undertale-save-converter#2]
Hello, good afternoon, I hope today's been kind to you!

I hope I am articulating this correctly, as it is my first time marking an issue...

I had used this & another Undertale converter, as I had trouble converting with this one, in order to use [Flowey's Time Machine](https://crumblingstatue.github.io/FloweysTimeMachine), for my Switch to get an item I forgot from Snowdin, then I converted from Switch to PC & then from PC to Switch,

The other converter is located here: 
• https://github.com/tomchapin/undertale-save-converter

For some reason, upon converting from PC back to Switch & on my Emummc (Virtual Switch using Hekate or Atmosphere), I am unable to stop or fix my HP, Gold, Attack, and Defense for both regular and weapon from being in the levels of thousands to tens of thousands, even after starting a new save file, redownloading the game, or using another profile, & here is a screenshot showing so:

![image](https://user-images.githubusercontent.com/73446603/97124130-1f690000-16ec-11eb-9bb2-30986b6f6113.png)

I do not have any cheats I am aware of activating, as EdiZon (a Switch cheats manager) says there are cheats available, though, none that match my version,

Also, here is a copy of my Undertale.sav/Undertale.ini & File 0, File 9, and Undertale.ini I pasted on Pastebin, with the files that I had converted, if it helps,

File 0, File 9, Undertale.sav & Undertale.ini 
• https://pastebin.com/QgnXYZVG

Tl;Dr: May I please know if there is any way to fix my Undertale Switch save on Emmumc, as I'd like to play with standard HP, Gold, Attack & Defense?

Thank you for your time, I appreciate it!

**Comment by infinitytrain** [comment]
Edit: I found out through trial-and-error and deleting the cheats file on my Switch that EdiZon was affecting the game despite saying otherwise!, and everything is fixed! 

Thank you for your time, I apologize for the misunderstanding, and thank you for the cool converter!
